exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -685,5 +685,6 @@ drill.exec.options: {
     exec.query.return_result_set_for_ddl: true,
     # ========= rm related options ===========
     exec.rm.queryTags: "",
-    exec.rm.queues.wait_for_preferred_nodes: true
+    exec.rm.queues.wait_for_preferred_nodes: true,
+    exec.statistics.tdigest_compression: 100
```

Comment:
Do we want to expose the algorithm name to the user? We had a similar discussion for HLL where we decided it would be better to have a generic name.

Reply:
The 'compression' is the name that t-digest uses for determining the number of centroids.  I cannot think of something more generic for this since if we change the underlying algorithm it may not use centroids at all.  I am thinking of keeping this as-is.  Let me know if you can think of a generic representation.  We could revisit the HLL names if we want to be consistent.